ncm-nfs/src/main/pan/components/nfs/schema.pan

```diff
@@ -15,7 +15,7 @@ type structure_nfs_exports = {
 type structure_nfs_mounts = {
     'device'     : string
     'mountpoint' : string
-    'fstype'     : string with match(SELF, '^nfs|none$')
+    'fstype'     : string with match(SELF, '^nfs|panfs|none$')
```

Comment:
you are aware that the string `"nfspanfsnone"` matches this regexp, right :smile: use
```pan
match(SELF, '^(nfs|panfs|none)$')
```

(pan really needs a `in` test)